core/src/main/java/com/redhat/lightblue/client/request/AbstractLightblueMetadataRequest.java

```diff
@@ -36,14 +54,7 @@ public String getRestURI(String baseServiceURI) {
     }
 
     @Override
-    public String getBody() {
-        JsonNode body = getBodyJson();
-        if (body == null) {
-            return null;
-        }
-        return body.toString();
+    public JsonNode getBodyJson() {
```

Comment:
abstract, instead of return null? could be, but then I have to implement it
in all the concrete classes.

On Fri, May 20, 2016 at 12:19 PM, Dennis Crissman <notifications@github.com>
wrote:

> In
> core/src/main/java/com/redhat/lightblue/client/request/AbstractLightblueMetadataRequest.java
> <https://github.com/lightblue-platform/lightblue-client/pull/293#discussion_r64085150>
> :
>
> > @@ -36,14 +54,7 @@ public String getRestURI(String baseServiceURI) {
> >      }
> >
> >      @Override
> > -    public String getBody() {
> > -        JsonNode body = getBodyJson();
> > -        if (body == null) {
> > -            return null;
> > -        }
> > -        return body.toString();
> > +    public JsonNode getBodyJson() {
>
> abstract?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/lightblue-platform/lightblue-client/pull/293/files/a26850a67fc0252d7f24a09ecf4c45fd80b40b2f#r64085150>
>